lib/scholar/covariance.ex

```diff
@@ -29,39 +29,61 @@ defmodule Scholar.Covariance do
   @doc """
   Computes covariance matrix for sample inputs `x`.
 
+  #{~S'''
+  The value on the position $(i, j)$ in matrix is caluculated using formula:
+  $$ Cov(X\_{i}, X\_{j}) = \frac{\sum\_{k}\left(x\_{k} - \bar{x}\right)\left(y\_{k} - \bar{y}\right)}{N - 1} $$
```

Comment:
```suggestion
  $$ Cov(X_i, X_j) = \frac{\sum_{k = 1}^{N}\left(x_k - \bar{x}\right)\left(y_k - \bar{y}\right)}{N - 1} $$
```

Reply:
This one does not work. For double dollars, you need to escape underscore. At least with configuration we have at this moment. 
![image](https://user-images.githubusercontent.com/56299341/210445475-db9f68e0-f83d-4132-9e95-a701b7c887c5.png)

Reply:
I see. Can you show the rendered version with the escaped underscore?

Reply:
![image](https://user-images.githubusercontent.com/56299341/210446938-1006544d-4ea1-4e14-bcc3-317b95994e13.png)